example/gan/CGAN_mnist_R/CGAN_train.R

```diff
@@ -64,32 +80,33 @@ input_names_D <- mxnet:::mx.model.check.arguments(D_sym)
 
 ###################################################
 #initialize optimizers
-optimizer_G<-mx.opt.create(name = "adadelta",
+optimizer_G <- mx.opt.create(name = "adadelta",
                            rho=0.92, 
```

Comment:
I still see some inconsistent format here and there. Please consider using tools like [formatR](https://yihui.name/formatr/) to clean up the style.